### ISSUE TITLE
Have you trained the model for  the UCF101 from scratch?

### ISSUE THREAD

**Issue by lixianhang** [irhumshafkat/R2Plus1D-PyTorch#5]
I just trained for 45 epochs and set LR 0.01 for UCF101, but I got the really terrible result. Firstly the training loss is stable around 4, but cannot get lower(because the dataset is small, it should be overfitted). And second, the test accuracy is around 1% even after 45 epochs. I followed your instruction to process the dataset. I don't know where is wrong. Could you please tell me your result if you have?

**Comment by qingxiaoli** [comment]
I have met the same problem. How did you solve the problem? Can you please share?

**Comment by yechanp** [comment]
Hello , it might be too late. 
I found that learning rate is decreasing too fast. Change gamma 0.1 to big number such as 0.9. It will be better
